### ISSUE TITLE
.bz2 .gz .xz? 

### ISSUE THREAD

**Issue by petrovag** [ProfessorNavigator/mylibrary#1]
Does you plan to support the fictionbook compressed .bz2 .gz .xz?

**Comment by ProfessorNavigator** [comment]
Yes, why not. But I don't know - when it will happen. I need to develop some other projects (not only programming).

**Comment by ProfessorNavigator** [comment]
https://github.com/ProfessorNavigator/mylibrary/releases/tag/v2.1
